### PR TITLE
Feature/response body

### PR DESCRIPTION
**Pull request by lunich**
Module instance variables are not thread-safe. Let's avoid them.

### CODE REVIEW

**Review by f1rstsense — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# 0.4.0
+
+* Add response body to Client
+* Configuration class removed. Now you need to configure an instance of Client class
+
+#0.3.0
```
Comment (sqbell) [inline CHANGELOG.md]: Space after `#`?

**Review by sqbell — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -18,25 +18,42 @@ Or install it yourself as:
...
 
-You can configurate by adding this code to Rails initializer for example:
+You can create and configurate a client:
 
 ```ruby
-Fortytwoish.configure do |config|
-  config.token = 'XXX'
-  config.encoding = 'GSM7' # GSM7, UCS2, BINARY are available, GSM7 is default
-end
+client = Client.new(
```
Comment (jonkerz) [inline README.md]: `Fortytwoish::Client`?

File: CHANGELOG.md (hunk 1 above)
Comment (lunich) [reply CHANGELOG.md]: Addressed

File: README.md (hunk 2 above)
Comment (lunich) [reply README.md]: Addressed

**Pull request merged**